cmd/promtool/main.go

```diff
@@ -640,7 +640,7 @@ func QueryLabels(url *url.URL, name string, start, end string, p printer) int {
 	// Run query against client.
 	api := v1.NewAPI(c)
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	val, warn, err := api.LabelValues(ctx, name, stime, etime)
+	val, warn, err := api.LabelValues(ctx, name, []string{}, stime, etime)
```

Comment:
Is that required? 🤔

Reply:
> LabelValues now requires a list of string matchers.

Got my answer, looks good then, but `nil` would do too.

Reply:
https://github.com/prometheus/client_golang/pull/828